lua/nvim-tree/git/utils.lua

```diff
@@ -9,7 +9,7 @@ function M.get_toplevel(cwd)
   local ps = log.profile_start("git toplevel %s", cwd)
   log.line("git", cmd)
```

Comment:
cmd is unused.

Please replace cmd with `{ "git", "-C", vim.fn.shellescape(cwd), "rev-parse", "--show-toplevel" }`, log it and and pass it to `vim.fn.system(`